wordpress-mobile/WordPress-Android
# Replace bottom bar with action mode icon in media picker

This PR introduces UI changes to the consolidated media picker. This includes 2 main things:
- The "Edit" action for images is now moved to the action mode toolbar. It's visible if only images are selected
- The Bottom bar is gone (so I've removed all the logic around it and most of the logic around icon clicks)

To test:
- Go to Gutenberg editor
- Add an image block
- Click on "Add media"
- Click on "Choose from device"
- The media picker is opened
- Notice there is no bottom bar
- Select an item
- Notice that there is an edit button and a confirm button in the action bar toolbar
- Test both work as expected

To test 2:
- Go to Gutenberg editor
- Add an video block
- Click on "Add media"
- Click on "Choose from device"
- The media picker is opened
- Notice there is no bottom bar
- Select an item
- Notice that there is a confirm button in the action bar toolbar and the edit button is not visible
- Test the confirm button works as expected

To test:
- Go to Gutenberg editor
- Add an gallery block
- Click on "Add media"
- Click on "Choose from device"
- The media picker is opened
- Notice there is no bottom bar
- Select a few items
- Notice that there is an edit button and a confirm button in the action bar toolbar
- Test both work as expected

PR submission checklist:
